card/db/migrate_core_cards/data/more_bootstrap.json

```diff
@@ -0,0 +1,51 @@
+{
+  "url": "http://localhost:3000/export.json",
+  "timestamp": "2015-02-25 14:55:17 +0100",
+  "card": {
+    "name": "export",
+    "type": "Pointer",
+    "content": "[[style: glyphicons]]\n[[simple skin]]\n[[simple bootstrap skin]]\n[[Default Layout]]\n[[*all+*script]]\n[[*page header]]\n[[style: bootstrap compatible]]\n[[Old Layout]]",
+    "value": [
```

Comment:
Do we have a definition of this json cards format somewhere on http://wagn.org/ ?  Can we use contextual names in *subcards*?  I'm guessing on a pointer, this actually creates the card content from the pointer names, but presumably you can put children here, or not?